docs/reference/policies/platforms.md

```diff
@@ -31,4 +31,21 @@ build on aarch64 Windows, but does not consider either platform to be supported
 minimum supported Windows versions are Windows 10 and Windows Server 2016, following
 [Rust's own Tier 1 support](https://blog.rust-lang.org/2024/02/26/Windows-7.html).
 
-uv supports and is tested against Python 3.8, 3.9, 3.10, 3.11, 3.12, and 3.13.
+## Python support
+
+uv supports and is tested against the following Python versions:
+
+- 3.8
+- 3.9
+- 3.10
+- 3.11
+- 3.12
+- 3.13
+
+uv supports the following Python implementations:
+
+- CPython
+- PyPy
+- GraalPy
+- Pyston
+- Pyodide
```

Comment:
There are different support levels as CPython we manage, PyPy and GraalPy we install, and Pyston and Pyodide we support, while I wouldn't consider problems with Pyston and Pyodide show-stoppers, Pyston being unmaintained and Pyodide being young and very different.